src/gui/search/pluginselectdlg.cpp

```diff
@@ -314,6 +315,16 @@ void PluginSelectDlg::finishAsyncOp()
         setCursor(QCursor(Qt::ArrowCursor));
 }
 
+void PluginSelectDlg::finishPluginUpdate()
+{
+    --m_pendingUpdates;
+    if (m_pendingUpdates == 0 && !m_updatedPlugins.isEmpty()) {
```

Comment:
In `pluginInstallationFailed()` and `pluginUpdateFailed()` only the counter is decremented. The failed plugin name isn't recorded.